lynx/common/actions/drop.py

```diff
@@ -34,21 +34,22 @@ def drop_in_overworld(self, scene: 'Scene', inventory: dict):
                 create_action = CreateObject(object_created.serialize())
                 scene.add_to_pending_actions(create_action.serialize())
 
-
     def apply(self, scene: 'Scene') -> None:
         agent: Object = scene.get_object_by_id(self.object_id)
         player_name = agent.owner
+        player = scene.get_player(agent.owner)
         if self.target_position == scene.get_drop_area_of_a_player(player_name):
             self.drop_in_drop_area(scene, player_name, agent.inventory)
         else:
             self.drop_in_overworld(scene, agent.inventory)
 
         agent.inventory = {}
+        player.get_agent_by_id(agent.id).drop_inventory()
```

Comment:
Czy ta zmiana dotyczy tego pobierania `details` o agentach?

Reply:
To jest po to żeby podejrzeć inventory agenta na froncie

Reply:
```
    def drop_inventory(self):
        self.inventory = {}
```
Tylko że ustawiamy inventory na pustą listę, czyli to nie będzie to samo jak `agent.inventory = {}`

Reply:
Jest tka samo, i też ustawiamy na pusty słownik tylko jest to opakowane w metodę